api/discovery/discovery.proto

```diff
@@ -48,7 +48,7 @@ service Discovery {
   };
 
   // Lists all evidences collected in the last run, exposed as REST.
-  rpc Query(QueryRequest) returns (QueryResponse) {
+  rpc Query(ListResourcesRequest) returns (QueryResponse) {
```

Comment:
One last thing: rename QueryResponse to `ListResourcesResponse` please and then we are ready to merge :)